Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiTable.c

```diff
@@ -434,7 +438,24 @@ PlatformUpdateAcpiTable (
   } else if (Table->Signature == EFI_BDAT_TABLE_SIGNATURE) {
     UpdateBdatAcpiTable (Table, LdrGlobal->FspHobList);
     DEBUG ((DEBUG_INFO, "Updated BDAT Table in AcpiTable Entries\n"));
-  } else if (Table->Signature == EFI_ACPI_6_1_LOW_POWER_IDLE_TABLE_STRUCTURE_SIGNATURE){
+  } else if (Table->Signature == EFI_ACPI_6_1_FIXED_ACPI_DESCRIPTION_TABLE_SIGNATURE) {
+    DEBUG ( (DEBUG_INFO, "Updated FADT Table entries in AcpiTable\n") );
+    {
+      EFI_ACPI_6_1_FIXED_ACPI_DESCRIPTION_TABLE *FadtPointer;

```

Comment:
Move local variable to the top of the function

Reply:
Noted and uploaded revised patch